desc/objectives/utils.py

```diff
@@ -92,6 +92,28 @@ def factorize_linear_constraints(objective, constraint, x_scale="auto"):  # noqa
         A = A[:, cols]
     assert A.shape[1] == xp.size
 
+    # check for degenerate rows and delete if necessary
+    row_idx_to_delete = np.array([], dtype=int)
+    for row_idx in range(A.shape[0]):
+        # find all rows equal to this row
+        rows_equal_to_this_row = np.where(np.all(A[row_idx, :] == A, axis=1))[0]
```

Comment:
I think you also need to check that b is the same for those rows, otherwise the constraints are incompatible. Could just add b as another column before doing this loop

Reply:
ah right, good catch